Tests/MongoSwiftSyncTests/CrudTests.swift

```diff
@@ -106,6 +106,13 @@ final class CrudTests: MongoSwiftTestCase {
             "unacknowledged-bulkWrite-update-hint-clientError": ["*"],
             "unacknowledged-bulkWrite-replace-hint-clientError": ["*"]
         ]
+        // Skipping due to a bug in server latest. TODO: SWIFT-1359 unskip
+        let client = try MongoClient.makeTestClient()
```

Comment:
we don't really have a way in the runner itself to do conditional skips based on server version/topology, it seems, besides editing changes into the JSON files which didn't seem like a great approach.